src/Compute/Compute/Generated/Disk/Config/NewAzureRmDiskConfigCommand.cs

```diff
@@ -265,6 +281,16 @@ private void Run()
                 EncryptionSettingsCollection = vEncryptionSettingsCollection,
             };
 
+            // this is to hide the breaking change for upload
+            if ("upload".Equals(vDisk.CreationData?.CreateOption?.ToLowerInvariant()))
```

Comment:
Should add `vDisk.CreationData?.UploadSizeBytes != null` in if statement?

Reply:
vDisk.CreationData?.UploadSizeBytes == null should be added.